cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,99 @@
-/// <reference types='cypress' />
+import { workerData } from '../support/userData';
+/// <reference types = "cypress" />
+
+function addWorker() {
+  const workerCount = 3;
+  Cypress._.times(workerCount, () => {
+    const newWorker = workerData();
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type(newWorker.firstName);
+    cy.get('#lastName').type(newWorker.lastName);
+    cy.get('#userEmail').type(newWorker.email);
+    cy.get('#age').type('25');
+    cy.get('#salary').type(newWorker.salary);
+    cy.get('#department').type(newWorker.department);
+    cy.get('#submit').click();
+
+    cy.get('.rt-tbody').contains(`${newWorker.firstName}`).should('exist');
+
+    return newWorker;
+  });
+}
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should add a new worker', () => {
+    addWorker();
+  });
+
+  it('should switch to pages by pagination', () => {
+    addWorker();
+
+    cy.contains('.-btn', 'Next').should('exist');
+    cy.contains('.-btn', 'Previous').should('exist');
+
+    cy.get('select[aria-label="rows per page"]').select('5');
+
+    cy.contains('.-btn', 'Next').click();
+    cy.get('input[type="number"]').should('have.value', 2);
+
+    cy.contains('.-btn', 'Previous').click();
+    cy.get('input[type="number"]').should('have.value', 1);
+  });
+
+  it('should delete a worker', () => {
+    cy.get('#delete-record-1').click();
+    cy.contains('div', 'cierra@example.com').should('not.exist');
+  });
+
+  it('should delete all workers', () => {
+    cy.get('#delete-record-1').click();
+    cy.contains('div', 'cierra@example.com').should('not.exist');
+
+    cy.get('#delete-record-2').click();
+    cy.contains('div', 'alden@example.com').should('not.exist');
+
+    cy.get('#delete-record-3').click();
+    cy.contains('div', 'kierra@example.com').should('not.exist');
```

Comment:
The test assumes that 'kierra@example.com' is present before deletion. Ensure that the test setup includes adding this worker or adjust the test to reflect the current state of the data.